### DISCUSSION TITLE
A Question about the first variable of the Ensemble Models

### DISCUSSION THREAD

**Discussion by shouyisxty** [open-meteo/open-meteo#366] [Q&A]
I have a question about the first variable of download data of Ensemble Models. For example, I select Ensemble Model "DWD Icon EPS Seamless" and hourly variable "Temperature (2m)".  The downloaded data include 40 variables. The first variable named "temperature_2m (¬∞C)",  and the remaining 39 variables are named "temperature_2m_member01 (¬∞C)" to "temperature_2m_member39 (¬∞C)".  From variable names, I think there are 39 members of models in the ensemble data. While the first variable "temperature_2m (¬∞C)" has no member. So if the first variable is the forecasts by a baseline model? or it is the actual weather data to compare with the forecasts data?  Hope to get some clarification on the first variable in ensemble forecast data. Thanks a lot!

**Comment by patrick-zippenfenig** [comment]
Correct, the first returned variable is the "control" forecast. All other members are perturbed forecasts where slight variations in the initial conditions had been introduced.

The control forecast is not the same as presented in the regular [weather Forecast API ](https://open-meteo.com/en/docs), because ensemble models use lower resolution and output fewer variables.
